sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.4 (2020-03-16)
+
+- `@azure/amqp-common` library has been updated to version 1.0.0-preview.11 with an improvement that increases the default timeout for the requests sent to the service from 10s to 60s to avoid `"$cbs" timeout`.
```

Comment:
```suggestion
- `@azure/amqp-common` library has been updated to version 1.0.0-preview.11 with an improvement that increases the default timeout for the authorization requests sent to the service from 10s to 60s to avoid frequent timeout errors.
```